livedata/src/main/java/io/getstream/chat/android/livedata/entity/ReactionEntity.kt

```diff
@@ -46,7 +46,7 @@ data class ReactionEntity(@PrimaryKey var messageId: String, var userId: String,
     var syncStatus: SyncStatus = SyncStatus.COMPLETED
 
     /** create a reactionEntity from a reaction object */
-    constructor(r: Reaction) : this(r.messageId, r.user!!.id, r.type) {
+    constructor(r: Reaction) : this(r.messageId, r.fetchUserId(), r.type) {
```

Comment:
seems `Reaction.fetchUserId()` function is not there in LLC. Is there a PR related to this change somewhere?

Reply:
its on the develop branch atm.